static/js/stream_popover.js

```diff
@@ -462,7 +461,11 @@ function build_move_topic_to_stream_popover(e, current_stream_id, topic_name) {
             },
             (xhr) => {
                 dialog_widget.hide_dialog_spinner();
-                show_error_msg(xhr.responseJSON.msg);
+                ui_report.error(
+                    $t_html({defaultMessage: "Error moving topic"}),
```

Comment:
It looks like this discards the (Translated) message we got from the server, which in theory could explain what the error is.  What's the thinking here?  I would have expected `xhr.responseJSON.msg` to be what we want, at least if there's a value there.  (I guess maybe this error handling may not deal well with the case where it's a client-side error, like networking?)

Reply:
It doesn't:
https://github.com/zulip/zulip/blob/b9629d72a92a9fce6a0393c0f31850a9d2f0e857/static/js/ui_report.ts#L44-L47

In case of network failure:
![image](https://user-images.githubusercontent.com/58626718/143437183-c2a23cb7-20be-428d-8b70-376b604c5870.png)